src/website/src/app/documentation/demos/datepicker/demos/datepicker-api.demo.html

```diff
@@ -5,9 +5,10 @@ <h4><code class="clr-code">clrDate</code> Directive</h4>
     <code class="clr-code">input</code> field. Then, place the input inside the
     <code class="clr-code">clr-date-container</code> container element.
 </p>
-<form class="datepicker-demo" clrForm clrLayout="vertical">
+<form class="datepicker-demo" clrForm clrLayout="horizontal">
     <clr-date-container>
-        <input type="date" clrDate name="demo" [(ngModel)]="demo">
+        <label for="basicDemo">Basic Demo</label>
```

Comment:
Need to remove `for` and `id` from all demos, sorry if that wasn't clear.

Reply:
I see, I will remove these as well.